deploy/operator/overlays/dev-template/kustomization.yaml

```diff
@@ -8,3 +8,14 @@ images:
   - name: hacbs-jvm-operator
     newName: quay.io/QUAY_USERNAME/hacbs-jvm-controller
     newTag: dev
+
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: BUILD_REQUEST_PROCESSOR_IMAGE
```

Comment:
So changing the name to `JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE` would allow you to already tie in with was our openshift CI integration is doing on PR e2e invocations @stuartwdouglas per https://github.com/redhat-appstudio/jvm-build-service/blob/main/.ci/common-ci-settings-and-functions.sh#L52